src/views/Home.jsx

```diff
@@ -9,6 +9,7 @@ export function Home({ data, setListPath }) {
 			<ul>
 				{data &&
 					data.map((list) => {
+						console.log('here is list path', list.path);
```

Comment:
```suggestion
```
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking ):** often good to clean up console logs before merging!